### PR TITLE
Added support for 8.282.08.1

### PR DESCRIPTION
**Pull request by mfuhrmann**
rel https://github.com/Proemion/ts2-psp-merge/issues/24955

Giving a try here. Let's see if that is what we need...

### CODE REVIEW

File: bin/list-all [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-echo "8.121 8.131 8.141 8.144 8.151 9.0.1 corretto-8.202.08.2 corretto-8.242.08.1 corretto-11.0.7.10.1"
+echo "8.121 8.131 8.141 8.144 8.151 9.0.1 corretto-8.202.08.2 corretto-8.242.08.1 corretto-11.0.7.10.1 coretto-8.282.08.1"
```
Comment (BaCaRoZzo) [inline bin/list-all]:
```suggestion
echo "8.121 8.131 8.141 8.144 8.151 9.0.1 corretto-8.202.08.2 corretto-8.242.08.1 corretto-11.0.7.10.1 corretto-8.282.08.1"
```

**Review by BaCaRoZzo — APPROVED** [review]

File: bin/list-all (hunk 1 above)
Comment (mfuhrmann) [reply bin/list-all]: :facepalm:

**Comment by mfuhrmann** [comment]
@BaCaRoZzo Anything else than `merge` here to do?

**Comment by BaCaRoZzo** [comment]
>@BaCaRoZzo Anything else than merge here to do?

No. If just has to be merged. Then the users can update the plugin and will see the new option to be downloaded.

Just in case I've double-checked the download and it works fine. I have a doubt though, probably pretty stupid but better double-check with @unicolet: how is `sigkey` defined? I see it is repeated for some options but not all are the same. :thinking:

**Review by unicolet — APPROVED** [review]
thanks!

@BaCaRoZzo AFAIR it comes from https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/downloads-list.html#signature

**Comment by mfuhrmann** [comment]
@unicolet @BaCaRoZzo Do we have to roll this update out somewhere after merging? Or is it done with merging?

**Comment by unicolet** [comment]
@mfuhrmann the process is as follows:

1. add support for a new asdf-managed java version in this plugin, merge pull request
2. repositories which want to use the new version will have to indicate so in their `.tool-versions` file, for example ts2-psp-merge has: https://github.com/Proemion/ts2-psp-merge/blob/master/.tool-versions
3. after editing `.tool-versions` every developer needs to actualize the changes using `asdf install`, or for ts2-psp-merge `b setup update`, see https://github.com/Proemion/ts2-psp-merge/blob/master/build_scripts/build.sh#L74

**Comment by unicolet** [comment]
@mfuhrmann you can read about how asdf works at: https://asdf-vm.com/#/

**Comment by mfuhrmann** [comment]
@unicolet @BaCaRoZzo Thanks for your help!

**Pull request merged**